packages/generator/src/options/options-parser.spec.ts

```diff
@@ -68,6 +76,27 @@ describe('options parser', () => {
       });
     });
 
+    it('coerces value', () => {
```

Comment:
[req] Add a test to ensure the desired behaviour for coerce with a property having a default value.

Reply:
Done, let me know what you think.